src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/virtual-machines-overview-tab-utilization.scss

```diff
@@ -0,0 +1,63 @@
+.VirtualMachinesOverviewTabUtilization--main {
+  .title {
```

Comment:
add margin bottom, or in another place.
add space between title and info

Reply:
you probably mean the subtitle , i spaced them :)